figures/palettes/oncoprint_color_palette.tsv

```diff
@@ -11,11 +11,13 @@ In_Frame_Ins	#FFE4E1
 Stop_Codon_Ins	#CC79A7
 Start_Codon_Del	#56B4E9
 Fusion	#7B68EE
-Multi_Hit	#00F021
+Multi_Hit	#CCCCCC
 Hom_Deletion	#313695
 Hem_Deletion	#abd9e9
-amplification	#c51b7d
-loss	#0072B2
-gain	#D55E00
+Del	#0072B2
+Amp	#D55E00
 High_Level_Gain	#FF0000
 Multi_Hit_Fusion	#CD96CD
+Intron	#5ea4fe
+5'Flank	#386db0
+3'Flank	#c51b7d
```

Comment:
```suggestion
5'Flank	#7CCBAC
3'Flank	#FD8344
```
new color suggestions - moving these two to Amp/Del, where I had meant them to be previously, but it looks like things got shuffled. Let's see what these look like 😬